src/common/components/userMatchModal/userMatchModal.component.js

```diff
@@ -157,6 +177,26 @@ class UserMatchModalController {
   onFailure () {
     this.$window.location = '/'
   }
+
+  back () {
+    if (this.questionIndex === 1) {
+      this.changeMatchState('identity')
+    } else {
+      this.questionIndex--
+    }
+  }
+
+  continueCheckout () {
+    this.$window.location = '/checkout.html'
+  }
+
+  goToOpportunities () {
+    this.$window.location = '/'
+  }
+
+  goToDashboard () {
```

Comment:
Might be worth renaming this to `goToGivingDashboard`. The profile page or the receipt page could be considered dashboards as well.